views.py

```diff
@@ -0,0 +1,169 @@
+from flask import render_template, session, redirect, request
+from datetime import datetime
+
+from init import db, app
+from forms import LoginForm, RegistrationForm, OrderForm
+from tables import User, Dish, Category, Order
+from words import correct_word
+
+
+@app.route("/")
+def render_main():
+    items = db.session.query(Dish)
+    categories = db.session.query(Category)
+    session["word"] = correct_word(len(session.get("cart", [])))
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    return render_template("main.html", categories=categories, items=items, dishes=session.get("cart", []), user=user)
+
+
+@app.route("/cart/", methods=["GET", "POST"])
+def render_cart():
+    flag = 0
+    form = OrderForm()
+    cart = session.get("cart", [])
+    use_it = set()
+    new_cart = []
+    for item in cart:
+        if item not in use_it:
+            new_cart.append(item)
+            use_it.add(item)
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    session["word"] = correct_word(len(session.get("cart", [])))
+    return render_template("cart.html", form=form, new_cart=new_cart, flag=flag, user=user)
+
+
+@app.route("/add/<item>/")
+def add_to_cart(item):
+    flag = 0
+    form = OrderForm()
+    items = db.session.query(Dish)
+    cart = session.get("cart", [])
+    cart.append(item)
+    session["cart"] = cart
+    for i in items:
+        if i.title == item:
+            if not session.get("price"):
+                session["price"] = 0
+            session["price"] += i.price
+    cart = session.get("cart", [])
+    use_it = set()
+    new_cart = []
+    for item in cart:
+        if item not in use_it:
+            new_cart.append(item)
+            use_it.add(item)
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    session["word"] = correct_word(len(session.get("cart", [])))
+    return render_template("cart.html", form=form, flag=flag, new_cart=new_cart, user=user)
+
+
+@app.route("/pop/<item>/")
```

Comment:
Maybe this should be HTTP DELETE method?

Reply:
Also, `pop` is operation without argument, removal of specific item is umm `remove`?)

Reply:
I agree, but let it be pop already))